### PR TITLE
Sf3 43/radio image styles

### PR DESCRIPTION
**Pull request by lowlandjuju**
Fixes SF3-43

Simple css styles added inline (I tried to use bootstrap classes but I think we might need to update to the latest version)

### CODE REVIEW

File: smartforests/templates/smartforests/include/radio_player.html [hunk 1]
```diff
@@ -21,9 +21,9 @@
     </div>
     <a class='text-decoration-none text-reset d-block position-relative' href='' data-smartforests-radio-episode-page-url>
       {% if episode.image %}
-          {% image episode.image height-48 data-smartforests-radio-episode-image=true %}
+          {% image episode.image fill-60x60 data-smartforests-radio-episode-image=true %}
       {% else %}
-          <img height="48" data-smartforests-radio-episode-image>
+          <img height="60" style="object-fit: cover; height: 75px; width: 75px;"  data-smartforests-radio-episode-image>
```
Comment (janbaykara) [inline smartforests/templates/smartforests/include/radio_player.html:26]: I don't know why I've never heard of `object-fit: cover`! Seems tidier than having to set up a background image.

**Review by janbaykara — APPROVED** [review]

**Comment by lowlandjuju** [comment]
Have added thumbnails to the model and a second imagechooser to the admin page.

**Review by janbaykara — APPROVED** [review]
Nice and simple.

One suggestion in case it takes your fancy: we could define getter properties for the Thumbnail that, if `thumbnail` doesn't exist in the database instead returns the `image` property - to maximise the likelihood of a pretty picture showing up. Not critical though.

**Comment by lowlandjuju** [comment]
@janbaykara Merging this as-is, although I think your suggestion is a good idea, I actually have a feeling the client won't be up for it, because it might obfuscate: the cropped backup image might make a user think that the correct thumbnail has already been added by someone. They really want separate images for the thumbnails, so think theyll be happier if the placeholder image shows, rather than a cropped banner image.  :smile:

**Pull request merged**